meetings/2022-11-16.md

```diff
@@ -45,45 +57,103 @@
+ > The foaf:Agent class is the class of agents; things that do stuff. A well known sub-class is foaf:Person, representing people. Other kinds of agents include foaf:Organization and foaf:Group.
+* SC: `<http://xmlns.com/foaf/0.1/Agent> <http://www.w3.org/2000/01/rdf-schema#comment> "An agent (eg. person, group, software or physical artifact)."` .
+* SC: In past meetings we discussed `acl:client` or `acl:agent` to refer to clients ( https://github.com/solid/web-access-control-spec/issues/81 ). I mentioned things that would be necessary to use `acl:agent` as an existing property, but the definition in natural language might need to be updated. Good news that `foaf:Agent` already covers it. It's feasible to just have `acl:agent` provided we update. `acl:agent` has range `foaf:Agent` only description doesn't cover agent. Mechanically it's fine.
+* HS: what this ShEx is doing is for person. you can have one for software agents and organizations. And these would have different shapes. 
+
+### Social profile as discovery mechanism
+URL: https://github.com/solid/webid-profile/issues/65
 
+* SC: Topic proposed by WT.
+* WT: I brought up that issue to indicate something that i find a bit strange and will influence data discovery.
+* WT: When we see a WebID profile doc, as used today, there's stuff that's in the shape we discussed today: names, addresses, etc. But on the other hand, we are defining mechanisms to let apps discover out data. For example, we have type indexes to point to where certain types of data are stores, and we have interop spec doing something similar with shape trees. My issue with this is once you start adding data to the profile doc, it is either duplicate data that is discoverable via other discovery mechanisms, in which case it has to be kept in sync, or only available in that profile doc, so the question is what data should be stored where? if I'm an app and i want your address, your photo album etc. do I look in your profile or use type indexes or interop registries? My intuition is that the profile document itself shouldn't contain any data, only pointers for discovery. that solves issues of duplicate data and where to find.
+* eP: I agree. Especially when we think about AuthZ, one case when data is public or protected info. The paper of Ruben Verborgh and team (ref???) using the case of birthday and so on, it is similar to email address, sometimes you want to share full name, first name... sometimes you want to make everything public. For that information AuthZ should happen before discovery. +1 to take advantage of existing discovery mechanisms to discover data. For different audiences you can select different. vcards they have access to instead of trying every seeAlso blindly. We need to work around some use cases from Ruben's paper and see how it can work, have different proposals. In general I agree discovery needs to be taken into, and preceded with AuthZ for protected information. 
```

Comment:
```suggestion
* eP: I agree. Especially when we think about AuthZ, one case when data is public or protected info. The paper of Ruben Verborgh and team (ref???) using the case of birthday and so on; it is similar to email address — sometimes you want to share full name, first name, etc., and sometimes you want to make everything public. For that information AuthZ should happen before discovery. +1 to take advantage of existing discovery mechanisms to discover data. For different audiences you can select different  vcards they have access to, instead of blindly trying every `seeAlso`. We need to work around some use cases from Ruben's paper and see how it can work, have different proposals. In general, I agree discovery needs to be taken into, and preceded with AuthZ for protected information. 
```